apps/ios/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "FluentUIReactNativeTester_iOS",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "react": "16.11.0",
+    "react-native": "0.62.2"
```

Comment:
Don't we want this to be 0.60 or so since our fork of react-native is what it should be consuming and that's on 0.60?

Reply:
Which leads to my dumb question - where do we specify we want react-native from the Microsoft fork?

Reply:
That'll have to come later (added it to the change notes). I haven't hooked it up yet which is great for iOS, but obviously is a requirement for macOS. A good question is do we even _want_ fluentui-rn for iOS to pull in our fork. Ideally our fork would only ship mac anyway, so long term we'd want to depend just on fb's implementation

Reply:
Started a conversation offline, but I think we definitely do to keep macOS and iOS aligned and because I'm concerned deforking entirely will increase our merge cost a trivial amount (assuming we resolve the major diffs between repos which we need to do anyway) at the expense of reducing the speed at which we can identify a fix in react-native needed in devmain and get it to devmain

Reply:
Yea I'm on board with that. Will plan to consume our fork of RN for both ios and macos